profiles/common/modules/custom/tmgmt_poetry/README.md

```diff
@@ -31,80 +31,44 @@ Table of content:
 
 # Installation
 ## Webmaster / Site builder:
-### Requesting access to poetry:
-:pray: Before you can start using poetry on Playground or Production, you should
-make a formal request to the Comm Europa Management team.
+### Requesting access to the DGT-Connector:
+:pray: Before you can start using the DGT-Connector on Playground or Production,
+a representative of your DG at the 
+[Europa Forum](http://www.cc.cec/home/europa-info/basics/management/committees/forum_europa/members/index_en.htm) you should make a formal request
```

Comment:
The "you" after the closing parenthesis of the Europa forum should be deleted, no?